### PR TITLE
Add Simplified Chinese translate for oauth2-provider

### PR DESCRIPTION
**Pull request by kasuganosoras**
Add Simplified Chinese translate for oauth2-provider

Link: https://docs.gitea.io/zh-cn/oauth2-provider/

### CODE REVIEW

File: docs/content/doc/development/oauth2-provider.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+目前 Gitea 只支持 [**Authorization Code Grant**](https://tools.ietf.org/html/rfc6749#section-1.3.1) 标准，并额外支持下列扩充标准：
+
+- [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636)
+- [OpenID Connect (OIDC)](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth)
+
+若想要让第三方应用使用 Authorization Code Grant，需要先在「设置」(`/user/settings/applications`)中注册一个新的应用程序。
+
+## Scope
+
+目前 Gitea 尚未支持 scope （参见 [#4300](https://github.com/go-gitea/gitea/issues/4300)），所有的第三方应用都可获得该用户以及他所属的组织中所有资源的读写权。
```
Comment (lunny) [inline docs/content/doc/development/oauth2-provider.zh-cn.md]: #4300 has been resolved in 1.19.0

File: docs/content/doc/development/oauth2-provider.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+   ![Authorization Page](/authorize.png)
+
+   用户将会被询问是否授权给您的应用程序。如果他们同意了授权，用户将会被重定向到 `REDIRECT_URL`，例如：
+
+   ```curl
+   https://[REDIRECT_URI]?code=RETURNED_CODE&state=STATE
+   ```
+
+1. 使用重定向提供的 `code`，您可以请求一个新的应用程序和 Refresh Token。Access Token Endpoint 接受 `application/json` 或 `application/x-www-form-urlencoded` 类型的 POST 请求，例如：
```
Comment (lunny) [inline docs/content/doc/development/oauth2-provider.zh-cn.md]: 2

File: docs/content/doc/development/oauth2-provider.zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+     "expires_in": 3600,
+     "refresh_token": "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJnbnQiOjIsInR0IjoxLCJjbnQiOjEsImV4cCI6MTU1NzgwNDMxMiwiaWF0IjoxNTU1MTc2MzEyfQ.S_HZQBy4q9r5SEzNGNIoFClT43HPNDbUdHH-GYNYYdkRfft6XptJBkUQscZsGxOW975Yk6RbgtGvq1nkEcklOw"
+   }
+   ```
+
+   `CLIENT_SECRET` 是生成给应用程序的唯一密钥。请记住，该密钥只会在您使用 Gitea 创建/注册应用程序后出现一次。如果您丢失了密钥，您必须在应用程序设置中重新生成密钥。
+
+   `access_token` 请求中的 `REDIRECT_URI` 必须符合 `authorize` 请求中的 `REDIRECT_URI`。
+
+1.发送 [API requests](https://docs.gitea.io/en-us/api-usage#oauth2) 时使用 `access_token` 以读写用户的资源。
```
Comment (lunny) [inline docs/content/doc/development/oauth2-provider.zh-cn.md]: 3

File: docs/content/doc/development/oauth2-provider.zh-cn.md (hunk 1 above)
Comment (kasuganosoras) [reply docs/content/doc/development/oauth2-provider.zh-cn.md]: A new commit was submitted, I translate it again from the latest en-us documents.

**Review by lunny — APPROVED** [review]

**Review by a1012112796 — APPROVED** [review]

**Pull request merged**